Sum_of_N_Natural_Numbers/Sum_of_N_Natural_Numbers.c

```diff
@@ -0,0 +1,23 @@
+/*
+    Sum of First N Natural Numbers
+
+    The task is to find sum of first n natural numbers, where n is 
+    the number input to us. We use the well known formula,
+    sum = n * (n + 1)/2 to compute the sum
+
+*/
+
+#include<stdio.h>
+
+int main() {
+    int num;
+    printf("Enter N : ");
```

Comment:
Remove space before `:` in all the files.

Reply:
Done. @SakshiShreya Please review.